pkg/constants/constants.go

```diff
@@ -0,0 +1,12 @@
+package constants
+
+// Log level constants for structured logging, starting from 1
+// 0 is default if no level is provided
+// Guidelines: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md#what-method-to-use
+const (
+	LvlDefault = iota + 1 // 1 - General configuration, routine logs
```

Comment:
Why the different prefix `Lvl` vs `Log`.

Idk how a go project is organized, but would it be better organized if you keep that in a more specific file `loglevel.go` then you can name the consts without prefix ?

Reply:
You are right we should keep it consistent.
I will change it to log.

From GO point of view, the package matters and not the file name.

Reply:
I changed it, thanks for the catch.